### ISSUE TITLE
Parsing Question

### ISSUE THREAD

**Issue by vukgogo** [peteroupc/CBOR-Java#10]
I have a question regarding parsing some values using this library. For example I have a hex string CBOR object that I Decode from Bytes. 

Eg:

`CBORObject messageObject = CBORObject.DecodeFromBytes(hexStringToByteArray(cborHex));`

And it parses fine and I can get some fields from it

`messageObject.get(2)` prints out `h'A2201865386350000102030405060708090A0B0C0D0E0F'`

Now I can see that `messageObject.get(2).getType()` returns ByteString

Is it possible to convert that to a map with this library. So if I paste that h'A22... value into the online CBOR parser it returns a map. See picture:

<img width="1443" alt="image" src="https://user-images.githubusercontent.com/16562675/87336010-3a35b100-c50f-11ea-8def-9fcf69c06406.png">

Basically trying to get that -1 and -100 values via this library. Thanks!

**Comment by vukgogo** [comment]
I solved it via: 

CBORObject messageObject2 = CBORObject.DecodeFromBytes(messageObject.get(2).GetByteString());

Which prints the correct value.
